lighthouse-core/audits/byte-efficiency/cache-headers.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const assert = require('assert');
+const parseCacheControl = require('parse-cache-control');
+const ByteEfficiencyAudit = require('./byte-efficiency-audit');
+const formatDuration = require('../../report/v2/renderer/util.js').formatDuration;
+const WebInspector = require('../../lib/web-inspector');
+const URL = require('../../lib/url-shim');
+
+// Ignore assets that have very high likelihood of cache hit
+const IGNORE_THRESHOLD_IN_PERCENT = 0.9;
+// Basically we assume a 10% chance of repeat visit.
+const PROBABILITY_OF_RETURN_VISIT = 0.1;
+
+class CacheHeaders extends ByteEfficiencyAudit {
+  /**
+   * @return {number}
+   */
+  static get PROBABILITY_OF_RETURN_VISIT() {
+    return PROBABILITY_OF_RETURN_VISIT;
+  }
+
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Caching',
+      name: 'cache-headers',
```

Comment:
hm if we go by consistency with the other byte efficiency audits they basically fall into either `<noun of thing being detected>` or `uses-<best practice we're encouraging>`

how about...

`uncached-assets`
`low-cache-ttl`
`uses-caching`
`uses-cache-headers`
`uses-long-cache-ttl`
?

Reply:
`uses-long-cache-ttl` certainly isn't exactly catchy but describes it well :) I like that since it's not just use, it's (if they're used) that they're long

Reply:
done